spec/apollo-federation/service_field_spec.rb

```diff
@@ -408,75 +406,64 @@ def self.visible?(context)
       )
     end
 
-    context 'with a filter' do
+    context 'context in schema generation' do
```

Comment:
This test broke in 1.13.x, because there's a [new validation](https://github.com/rmosolgo/graphql-ruby/blob/a7daf8fc1fc6e626a8597c4a5e978fddf1fdafb9/lib/graphql/static_validation/rules/query_root_exists.rb#L8) in GraphQL Ruby that warns against having an empty schema, but we were asserting that "Field '_service' doesn't exist on type 'Query'". 

After looking at the [original PR](https://github.com/Gusto/apollo-federation-ruby/pull/50) that introduced this, I think the intent is to make sure that `Schema.federation_sdl` respects context and uses it generation. 

See https://graphql-ruby.org/authorization/visibility.html

I struggled with fixing it trying to get the right indirect error message across different GraphQL versions. Also, I think schema level filters are deprecated? I can't find docs on it anymore on [graphql-ruby](https://github.com/rmosolgo/graphql-ruby). 

So with that, I rewrote the test to basically assert that we're using the context in SDL generation. Further improvements welcome.